hbase-server/src/main/java/org/apache/hadoop/hbase/mob/MobFileCleanerChore.java

```diff
@@ -79,7 +76,12 @@ public MobFileCleanerChore(HMaster master) {
           MobConstants.DEFAULT_MOB_CLEANER_PERIOD),
         TimeUnit.SECONDS);
     this.master = master;
-    cleaner = new ExpiredMobFileCleaner();
+    try (Connection conn = ConnectionFactory.createConnection(master.getConfiguration())) {
```

Comment:
this will call `conn.close` at the end of the try block.

Can we use `master.getConnection()` and just make use of the shared connection rather than making a new one?